controllers/user.js

```diff
@@ -66,31 +66,43 @@ class Users {
     * @param {object} res - The response object.
     * @param {object} next - The response object.
     */
-  static login(req, res, next) {
-    passport.authenticate('local', { session: false }, (
-      err,
-      user
-    ) => {
-      if (err) {
-        return next(err);
-      }
+  static async login(req, res) {
+    const usernameOrEmail = (req.body.email)
```

Comment:
The logic here is not quite accurate. Yes, you should get two properties from `req.body` which may be (as you have tried to) ``usernameOrEmail` and then `password`. 

At the point of finding the user with `User.findOne`, what you'd be doing is finding the user when `username` is equal to `req.body.usernameOrEmail` or email is equal to `req.body.usernameOrEmail`.

You'd have to use sequelize operators to accomplish this.

@rajeman already used one sequelize operator in the user validator, so he might be able to provide some context. You'd use two in your case; the `or` operator and the `iLike` operator.

Check sequelize documentation for the use of operators. Also, pair with @rajeman to implement this.

Reply:
ok... will do